code/game/gamemodes/caretaker/caretaker.dm

```diff
@@ -0,0 +1,9 @@
+/datum/game_mode/caretaker
```

Comment:
Hard no on that.
I dont see any way how that is sufficient to bring enough interesting gameplay to 10+ people.

The only way I could see that work out is as a addon-antagonist for other gamemodes.